packages/xo-web/src/xo-app/backup-ng/new/utils.js

```diff
@@ -9,3 +11,34 @@ export const getRandomId = () =>
   Math.random()
     .toString(36)
     .slice(2)
+
+export const FormFeedback = ({
+  showError,
+  error,
+  component: Component,
+  ...props
+}) => (
+  <div>
+    <Component
+      {...props}
+      style={
+        showError === undefined
+          ? undefined
+          : {
+              borderColor: showError ? 'red' : 'green',
+            }
```

Comment:
Don't you want to merge it with `props.style` so we can be able to style the component?